tests/ha/multi_pod_failure/test_multi_server_pod_failure.py

```diff
@@ -2095,116 +1870,79 @@ def test_deletes_during_kserver_pods_down(self):
-        LOGGER.debug("Response: %s", resp)
-        assert_utils.assert_true(resp[0], resp)
-        LOGGER.info("Step 5: Services of remaining pods are in online state")
-        event.clear()
-
+        self.restore_method = const.RESTORE_DEPLOYMENT_K8S
+        self.restore_pod = True
+        assert_utils.assert_true(resp[0], "Cluster/Services status is not as expected")
+        LOGGER.info("Step 2: Successfully shutdown random %s (K) server pod %s one by one during "
+                    "WRITEs in background. Verified cluster and services states are as expected &"
+                    " remaining pods status is online.", self.kvalue, self.pod_name_list)
         LOGGER.info("Joining background thread. Waiting for %s seconds to "
                     "collect the queue logs", HA_CFG["common_params"]["60sec_delay"])
         thread.join()
         responses = dict()
         while len(responses) != 2:
             responses = output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
-        LOGGER.info("Step 6: Verify status for In-flight WRITEs while %s (K) server pods going "
+        LOGGER.info("Step 3: Verify status for In-flight WRITEs while %s (K) server pods going "
                     "down should be failed/error.", self.kvalue)
         pass_logs = list(x[1] for x in responses["pass_res"])
         LOGGER.debug("Pass logs list: %s", pass_logs)
         fail_logs = list(x[1] for x in responses["fail_res"])
         LOGGER.debug("Failed logs list: %s", fail_logs)
         resp = self.ha_obj.check_s3bench_log(file_paths=pass_logs)
-        assert_utils.assert_false(len(resp[1]), f"Logs which contain failures: {resp[1]}")
+        assert_utils.assert_false(len(resp[1]),
+                                  f"Expected Pass, But Logs which contain failures: {resp[1]}")
         resp = self.ha_obj.check_s3bench_log(file_paths=fail_logs, pass_logs=False)
         assert_utils.assert_true(len(resp[1]) <= len(fail_logs),
-                                 f"Logs which contain pass: {resp[1]}")
-        LOGGER.info("Step 6: Verified status for In-flight WRITEs while %s (K) server pods "
+                                 f"Logs which contain passed IOs: {resp[1]}")
+        LOGGER.info("Step 3: Verified status for In-flight WRITEs while %s (K) server pods "
                     "going down.", self.kvalue)
-
-        LOGGER.info("Step 7: Create multiple buckets and run IOs")
+        LOGGER.info("Step 4: Create IAM user with multiple buckets and run IOs")
+        users = self.mgnt_ops.create_account_users(nusers=1)
+        self.s3_clean.update(users)
         self.test_prefix = 'test-40575-1'
         resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
                                                     log_prefix=self.test_prefix, nsamples=2,
-                                                    nclients=2)
+                                                    nclients=2, setup_s3bench=False)
```

Comment:
lets add skipcleanup=True for tests related to just READ/WRITEs to avoid any issues with deletes